project/Dependencies.scala

```diff
@@ -68,7 +68,8 @@ object Dependencies {
     filters,
     ws,
     "com.typesafe.akka" %% "akka-testkit" % "2.4.10" % Test,
-    "org.gnieh" %% "diffson" % "2.0.2" % Test
+    "org.gnieh" %% "diffson" % "2.0.2" % Test,
+    "de.leanovate.play-mockws" %% "play-mockws" % "2.5.1" % Test
```

Comment:
Could you achieve the same using the built-in `play.core.server.Server` https://www.playframework.com/documentation/2.5.x/ScalaTestingWebServiceClients#extracting-setup-code